language-plutus-core/common/Common.hs

```diff
@@ -19,32 +19,32 @@ import           System.FilePath            ((</>))
 import           Test.Tasty
 import           Test.Tasty.Golden
 
--- | A 'TestTree' of golden tests stored in some folder.
-type TestNestedGolden = Reader [String] TestTree
+-- | A 'TestTree' of tests stored in some folder.
+type TestNested = Reader [String] TestTree
```

Comment:
Are there any tests stored in a folder that are not golden? The `[String]` part specifies just a sequence of names, so we indeed can generically call this `TestNested`, but then I guess we should drop file system terms as well.

Reply:
Sure - having a nested name makes sense in other cases too.

Reply:
I just noticed that there was not in fact anything golden-specific about these functions, and removing "golden" from all the names removes some noise.

Reply:
I mean that in this comment:

> A 'TestTree' of tests stored in some folder.

which talks about folders while the type itself is generic and the concept of the "folder" can be irrelevant. I like the generalization, just let's adapt comments as well.

Reply:
Done.